packages/media-utils/package.json

```
@@ -0,0 +1,35 @@
+{
+	"name": "@wordpress/media-utils",
```

How about:

```suggestion
	"name": "@wordpress/media-upload",
```

As of today, this package will contain only:
- `MediaUpload` component
- `mediaUpload` callback.

The description also highlights that the purpose of this package is strictly related to uploading functionalities.

---

Hi @gziolo the mediaUpload function was renamed to uploadMedia as @noisysocks suggested. Do you think it still makes sense to rename the package or now that the function has a different name the current package name makes more sense?

---

Nice, this updated function name reads better somehow 😃 Regarding the name of the package, it’s your call. We don’t have other media utils at the moment and it’s hard to tell what future will bring. Ideally, we could call it media but I guess it isn’t possible because of backward compatibility considerations.

---

> Nice, this updated function name reads better somehow

It's because function names read nicer when they're a verb or begin with a verb 🙂